city_scrapers/spiders/il_police_professionalism.py

```diff
@@ -20,46 +20,70 @@ def parse(self, response):
+            'a[(.//*|.)[contains(text(),"Police Professionalism")]]/@href'
+        ).extract()
+        # titles = relavent_links = list_items.xpath(
+        #     'a[(.//*|.)[contains(text(),"Police Professionalism")]]/text'
+        # ).extract()
+        for i in range(0, len(relavent_links)):
+            link = response.urljoin(relavent_links[i])
+            yield response.follow(link, self._parse_item)
+
+    def _parse_item(self, response):
+        """`_parse_item` should always `yield` Meeting items"""
+        paragraphs = response.xpath('//p')
+        title = response.xpath('//h3/text()').get()
+        meeting = Meeting(
+            title=self._parse_title(response),
+            description=self._parse_description(response, paragraphs),
+            classification=self._parse_classification(title),
+            start=self._parse_start(response, paragraphs),
+            end=None,
+            time_notes='',
+            all_day=False,
+            location=self._parse_location(response),
+            links=self._parse_links(response),
+            source=response.url,
+        )
+        meeting['id'] = self._get_id(meeting)
+        meeting['status'] = self._get_status(meeting, title)
+        return meeting
 
     def _parse_title(self, item):
         """Parse or generate meeting title."""
         return "Commission on Police Professionalism"
 
-    def _parse_description(self, item):
+    def _parse_description(self, item, paragraphs):
+        return_list = []
+        # paragraphs = paragraphs[0:len(paragraphs) - 1]
+        for i in paragraphs:
+            xpath_selector = i.xpath('text()').get()
+            if xpath_selector is not None:
+                return_list.append(xpath_selector.strip().replace(u'\xa0', u''))
+        final_description = " ".join(return_list)
         """Parse or generate meeting description."""
-        return ""
+        return final_description
 
     def _parse_classification(self, item):
         """Parse or generate classification from allowed options."""
         return COMMISSION
 
-    def _parse_start(self, item):
+    def _parse_start(self, item, paragraphs):
         """Parse start datetime as a naive datetime object."""
-        title = item.xpath('text()').get()
-        date = "".join(title[title.index('(') + 1:title.index(')')].split(',')[1:])
-        date = date.strip(' ')
-        date = date.replace('p.m.', 'PM')
-        date = date.replace('a.m.', 'AM')
-        finalDate = datetime.strptime(date, "%B %d %Y %H:%M %p")
-        return finalDate
+        for p in paragraphs.getall():
+            date_match = re.search(r"[a-zA-Z]{3,10} \d{1,2}([a-z]{2})?,? \d{4}", p)
+            time_match = re.search(r"\d{1,2}(:\d{2})? ?[apm\.]{2,4}", p)
+            if date_match:
+                final_date = date_match.group(0)
+            if time_match:
+                final_time = time_match.group(0)
+        final_time = final_time.replace('a.m.', 'AM')
```

Comment:
A quicker way of doing this would be to just replace periods and make everything upper case
```python
final_time = time_match.group(0).replace('.', '').upper()
```

You also might want to set a default value for time like `final_time = '12:00AM' at the beginning of the function so that it won't error out if a time isn't found